doc/api/buffer.md

```diff
@@ -175,8 +175,8 @@ string into a `Buffer` as decoding.
   encoding a `Buffer` to a string, this encoding will omit padding.
 
 * `'hex'`: Encode each byte as two hexadecimal characters. Data truncation
-  may occur when decoding strings that do exclusively contain valid hexadecimal
-  characters. See below for an example.
+  may occur when decoding strings that do not exclusively consist of an even
+  number of hexadecimal characters. See below for an example.
```

Comment:
Optional nit-pick suggestion: The Microsoft style guide we follow suggests `might` instead of `may` in this context. But I wonder if it's actually `will occur` rather than `might occur`? All told:

```suggestion
* `'hex'`: Encode each byte as two hexadecimal characters. Data is truncated
  when decoding strings that do not exclusively consist of an even
  number of hexadecimal characters. See below for an example.
```